src/main.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+
+
+import '../style/main.scss';
+
+;let searchFormBoard = 'cats';
+let searchFormLimit = 5;
+
+const apiUrl = `http://reddit.com/r/${searchFormBoard}.json?limit=${searchFormLimit}`;
+
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  render() {
+    return (
+      <div>
+        <h3>Search Form</h3>
+        <form onSubmit={this.handleSubmit()}>
+          <input
+          type='text'
+          name='redditBoard'
+          placeholder='Search for a Reddit board'
+          />
+          <button type='submit'>Submit</button>
+        </form>
+      </div>
+    );
+  }
+
+  handleSubmit() {
```

Comment:
Our `handleSubmit` methods always take in an `event` and you would call `event.preventDefault()` like so:


handleSubmit(event){
   event.preventDefault()
// logic here
}